Makefile

```diff
@@ -225,6 +226,13 @@ test-module:
 			--rootdir=$(SRCDIR) \
 			-v
 
+.PHONY: test-stage
```

Comment:
In addition to the `stage/test` we now also have `{input,source}/test` folders. I guess the stage ones are the most important ones but I do wonder if we want targets for all of them or if we should just re-purpose `test-module` to just also run {source,stage,input}/test` tests?